content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -68,6 +68,22 @@ or contain bugs.
 
 Skipped CSVs do not need to be present in a catalog or set of manifests prior to adding to a catalog.
 
+#### Example for replaces and skips
```

Comment:
WDYT about `Example using replaces and skips`?

Reply:
Sounda perfect!

Reply:
Also, I doubt we have any real consistency throughout this set of documentation, but I was going through the existing content and it looks like we usually use a header for examples, but the header's name is in the form of `Example: <...>` like it is in https://deploy-preview-131--operator-lifecycle-manager.netlify.app/docs/tasks/install-operator-with-olm/.

Reply:
Modified.

Reply:
Yes, Example is shown as you explained. I modified according to it.